docs/guides/data-binding/how-to-bind-tabs.md

```diff
@@ -49,7 +49,7 @@ Finally create a `TabControl` and bind its `ItemsSource` property to the DataCon
       <DataTemplate>
         <DockPanel LastChildFill="True">
           <TextBlock Text="This is content of selected tab" DockPanel.Dock="Top" FontWeight="Bold" />
-          <TextBlock Text="{Binding Content}" />
+          <TextBlock Text="{Binding Content DataType="TabItemModel"}" />
```

Comment:
DataType should go into the DataTemplate-node above imo

Reply:
Ah yes, that's where it goes, not enough caffeine first thing in the morning here. I'll fix it.

Reply:
Sorry, still wrong